src/ts/helpers/svg.ts

```diff
@@ -189,3 +191,32 @@ export function removeClass(el: SVGElement, className: string) {
   }
   return el;
 }
+
+/**
+ * Recursivly finds the the base SVG Dom element of `el`
+ * @param  {Element} el - DOM element used as base for traversal
+ * @returns SVGSVGElement
+ */
+export function getBaseEl(el: Element): SVGSVGElement {
+  if (!el) {
+    throw Error("Could not find base element");
+  }
+  if (el.classList.contains(baseCssClassName)) {
+    return el as SVGSVGElement;
+  }
+  return getBaseEl(el.parentElement) as SVGSVGElement;
```

Comment:
But it's just the result of `getBaseEl(el.parentElement)` that gets casted as `SVGSVGElement ` - and the result is always a `SVGSVGElement` - else it throws an error. Though in fairness we could omit it, since the function signature takes care of this.